src/components/FilterDropdown.jsx

```diff
@@ -16,15 +18,18 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 function FilterDropdown(props) {
-  const { label, items, defaultValue, removeNone, onChange } = props;
-  const [choice, setChoice] = useState(defaultValue);
+  const { label, items, value, removeNone, onChange } = props;
   const classes = useStyles();
+  const location = useLocation();
+  const params = queryString.parse(location.search, { parseNumbers: true });
+  const sortByParams =
+    params.sortBy === undefined ? 'deadlineAsc' : params.sortBy;
```

Comment:
This component should not be concerning itself over sorting dropdown specific logic. This is a generic dropdown component that can used as a dropdown for **anything**.